libs-scala/nonempty/src/main/scala/nonempty/NonEmpty.scala

```diff
@@ -185,6 +185,8 @@ object NonEmptyColl extends NonEmptyCollInstances {
       (self: ESelf) to factory
     )
     def zipWithIndex: NonEmpty[CC[(A, Int)]] = un((self: ESelf).zipWithIndex)
+    def ++[B >: A](suffix: imm.Iterable[B]): NonEmpty[CC[B]] = un((self: ESelf) ++ suffix)
```

Comment:
You're right. `Map Ops` `++` is now ambiguous. I've moved `NEPreservingOps` into the `NonEmptyCollInstances` trait so that `Map Ops` gets higher priority.